adafruit_ble_berrymed_pulse_oximeter/adafruit_ble_transparent_uart.py

```diff
@@ -66,7 +72,7 @@ def read(self, nbytes=None):
         """
         return self._rx.read(nbytes)
 
-    def readinto(self, buf, nbytes=None):
+    def readinto(self, buf: bytes, nbytes: Optional[bytes] = None) -> Optional[int]:
```

Comment:
Sorry, missed this the first time!  So `readinto` acutally stores the data in the buffer provided.  So what we need is something that implements the buffer protocol and is mutable.  `circuitpython_typing` defines this as the type `WriteableBuffer` (the mutable counterpart to `ReadableBuffer`).  So you can import it at the top, and type `buf` as `WriteableBuffer`.

Only other thing is that `nbytes` is actually the _number_ of bytes to read.  If not provided, it does all that it can.  So you can type it as `int`!

Reply:
oh whoops, thanks for catching these